common/util/bbb.ts

```diff
@@ -220,26 +215,17 @@ export async function createOrUpdateCourseAttendanceLog(pupil: Pupil, ip: string
     const entityManager = getManager();
     const transactionLog = getTransactionLog();
     const courseAttendanceLog = new CourseAttendanceLog();
-    const now = new Date();
+
     if (subcourseId == null) {
         logger.error("Can't save new course attendance: subcourseId is null");
         logger.debug(courseAttendanceLog);
     } else {
         const activeLecture = await getActiveLectureOfSubcourse(subcourseId);
         if (activeLecture) {
             const logToUpdate = await getCourseAttendanceLog(activeLecture.id, pupil.id);
-            // Update log
             if (logToUpdate) {
-                // To prevent a high attendance time through rejoining check the absence time
-                // If absence time (difference between now and updatedAt) is shorter than interval time, add abscence time to attended time
-                // Else add courseAttendanceLogInterval to attendedTime
-                const absenceTime = now.getTime() - logToUpdate.updatedAt.getTime();
-                if (absenceTime < courseAttendanceLogInterval) {
-                    logToUpdate.attendedTime += absenceTime;
```

Comment:
Why is this check removed?

Reply:
Thank you! 
You're totally right. I can't remember why I deleted this part. I re-implemented this part and tested it. Should work like before, but it would be good if you could test it too, so there will be no surprise in the future :)